packages/protocol/test/gas.js

```diff
@@ -5,6 +5,7 @@ const getEvent = (receipt, event, arg) => { return receipt.logs.filter(l => l.ev
 const { deploy } = require('./helpers/deploy')(artifacts)
 const { DEFAULT_STAKE_AMOUNT, DEFAULT_LOCK_AMOUNT, EMPTY_DATA, ZERO_ADDRESS, ACTIVATED_LOCK } = require('./helpers/constants')
 
+// Note: should these be unskipped?
```

Comment:
I’m actually going to remove them as they are broken and don’t have any kind of assertion. They were only logging values on console for initial benchmarks.
We can always use eth-gas-reporter or something like that.